packages/password/tests/generate.test.js

```diff
@@ -1,16 +1,22 @@
-const { constants, _selectPasswordRules, HostnameInputError, ParserError, generate } = require('../index.js')
+const {
+    constants,
+    _selectPasswordRules,
+    HostnameInputError,
+    ParserError,
+    generate
+} = require('../index.js')
 const vendorRules = require('../rules.json')
 const fc = require('fast-check')
-const {Password} = require('../lib/apple.password.js')
+const { Password } = require('../lib/apple.password.js')
 
 function testUniqueTimes (domain, passwordRules, num = 10) {
     const pws = []
     for (let i = 0; i < num; i++) {
-        // these 3 domains have rulesets so weak that collisions are likely
+    // these 3 domains have rulesets so weak that collisions are likely
```

Comment:
Time to revisit adding prettier? I had a PR a loooong while ago, can't recall why I didn't go through with it.